[run] Environment variables are badly parsed, resulting in a panic

└> scalingo -a app run -e rf bash

```
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/Scalingo/cli/apps.buildEnv(0xc208028e50, 0x1, 0x1, 0x2, 0x0, 0x0)
	/home/leo/Projects/Go/src/github.com/Scalingo/cli/apps/run.go:143 +0x351
github.com/Scalingo/cli/apps.Run(0xc20802900a, 0x5, 0xc20800a060, 0x1, 0x1, 0xc208028e50, 0x1, 0x1, 0xab8810, 0x0, ...)
	/home/leo/Projects/Go/src/github.com/Scalingo/cli/apps/run.go:55 +0x152
github.com/Scalingo/cli/cmd.func·033(0xc208032820)
	/home/leo/Projects/Go/src/github.com/Scalingo/cli/cmd/run.go:52 +0x21b
github.com/Scalingo/cli/Godeps/_workspace/src/github.com/Scalingo/codegangsta-cli.Command.Run(0x886550, 0x3, 0x885cb0, 0x1, 0x8d2cb0, 0x1c, 0x959710, 0x3bb, 0x88e270, 0xe, ...)
	/home/leo/Projects/Go/src/github.com/Scalingo/cli/Godeps/_workspace/src/github.com/Scalingo/codegangsta-cli/command.go:105 +0xd6a
github.com/Scalingo/cli/Godeps/_workspace/src/github.com/Scalingo/codegangsta-cli.(*App).Run(0xc208054620, 0xc20800a000, 0x7, 0x7, 0x0, 0x0)
	/home/leo/Projects/Go/src/github.com/Scalingo/cli/Godeps/_workspace/src/github.com/Scalingo/codegangsta-cli/app.go:165 +0xe60
main.main()
	/home/leo/Projects/Go/src/github.com/Scalingo/cli/scalingo/main.go:102 +0xb13

goroutine 5 [chan receive]:
github.com/Scalingo/cli/Godeps/_workspace/src/github.com/stvp/rollbar.func·001()
	/home/leo/Projects/Go/src/github.com/Scalingo/cli/Godeps/_workspace/src/github.com/stvp/rollbar/rollbar.go:74 +0x53
created by github.com/Scalingo/cli/Godeps/_workspace/src/github.com/stvp/rollbar.init·1
	/home/leo/Projects/Go/src/github.com/Scalingo/cli/Godeps/_workspace/src/github.com/stvp/rollbar/rollbar.go:78 +0x5f

goroutine 7 [syscall]:
os/signal.loop()
	/opt/go/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/opt/go/src/os/signal/signal_unix.go:27 +0x35

goroutine 8 [chan receive]:
github.com/Scalingo/cli/signals.Handle()
	/home/leo/Projects/Go/src/github.com/Scalingo/cli/signals/handler.go:17 +0x265
created by main.main
	/home/leo/Projects/Go/src/github.com/Scalingo/cli/scalingo/main.go:90 +0x9d5

goroutine 12 [IO wait]:
net.(*pollDesc).Wait(0xc20800a300, 0x72, 0x0, 0x0)
	/opt/go/src/net/fd_poll_runtime.go:84 +0x47
net.(*pollDesc).WaitRead(0xc20800a300, 0x0, 0x0)
	/opt/go/src/net/fd_poll_runtime.go:89 +0x43
net.(*netFD).Read(0xc20800a2a0, 0xc20800f000, 0x1000, 0x1000, 0x0, 0x7f59a0c25cf0, 0xc208029258)
	/opt/go/src/net/fd_unix.go:242 +0x40f
net.(*conn).Read(0xc208038068, 0xc20800f000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/opt/go/src/net/net.go:121 +0xdc
net/http.noteEOFReader.Read(0x7f59a0c27560, 0xc208038068, 0xc208064108, 0xc20800f000, 0x1000, 0x1000, 0x7a89e0, 0x0, 0x0)
	/opt/go/src/net/http/transport.go:1270 +0x6e
net/http.(*noteEOFReader).Read(0xc20801cc80, 0xc20800f000, 0x1000, 0x1000, 0xc208010000, 0x0, 0x0)
	<autogenerated>:125 +0xd4
bufio.(*Reader).fill(0xc20805a7e0)
	/opt/go/src/bufio/bufio.go:97 +0x1ce
bufio.(*Reader).Peek(0xc20805a7e0, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
	/opt/go/src/bufio/bufio.go:132 +0xf0
net/http.(*persistConn).readLoop(0xc2080640b0)
	/opt/go/src/net/http/transport.go:842 +0xa4
created by net/http.(*Transport).dialConn
	/opt/go/src/net/http/transport.go:660 +0xc9f

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/opt/go/src/runtime/asm_amd64.s:2232 +0x1

goroutine 13 [select]:
net/http.(*persistConn).writeLoop(0xc2080640b0)
	/opt/go/src/net/http/transport.go:945 +0x41d
created by net/http.(*Transport).dialConn
	/opt/go/src/net/http/transport.go:661 +0xcbc
```

--- a/apps/run.go
+++ b/apps/run.go
@@ -140,7 +140,7 @@ func buildEnv(cmdEnv []string) (map[string]string, error) {
 
 	for _, cmdVar := range cmdEnv {
 		v := strings.Split(cmdVar, "=")
-		if len(v[0]) == 0 || len(v[1]) == 0 {
+		if len(v) != 2 || len(v[0]) == 0 || len(v[1]) == 0 {
 			return nil, fmt.Errorf("Invalid environment, format is '--env VARIABLE=value'")
 		}
 		env[v[0]] = v[1]

